src/main/java/seedu/address/model/person/predicates/CapContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.person.predicates;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.model.person.Person;
+
+
+/**
+ * Tests that a {@code Person}'s {@code Cap} matches any of the keywords given.
+ */
+public class CapContainsKeywordsPredicate implements Predicate<Person> {
+    private final List<String> keywords;
+
+    public CapContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        return keywords.stream()
+                .anyMatch(keyword -> Double.parseDouble(keyword) == (person.getCap().value));
```

Comment:
It might return an error if the keyword is not a double.
e.g. `find c/3.50/` and `find c/asa.`